MC/config/common/external/generator/TPCLoopers.C

```diff
@@ -390,6 +397,49 @@ FairGenerator *
     gauss = gSystem->ExpandPathName(gauss.c_str());
     scaler_pair = gSystem->ExpandPathName(scaler_pair.c_str());
     scaler_compton = gSystem->ExpandPathName(scaler_compton.c_str());
+    const std::array<std::string, 2> models = {model_pairs, model_compton};
+    const std::array<std::string, 2> local_names = {"WGANpair.onnx", "WGANcompton.onnx"};
+    const std::array<bool, 2> isAlien = {models[0].starts_with("alien://"), models[1].starts_with("alien://")};
+    const std::array<bool, 2> isCCDB = {models[0].starts_with("ccdb:"), models[1].starts_with("ccdb:")};
+    if (std::any_of(isAlien.begin(), isAlien.end(), [](bool v) { return v; }))
+    {
+        TGrid::Connect("alien://");
```

Comment:
Maybe it's not necessary to always connect to the GRID. This should only be needed in case of files on AliEn, right?

Reply:
This is already the case. The connection is opened only if at least an alien path is passed as an argument

Reply:
@jackal1-66 please always check for the existing gGrid before connecting, see https://github.com/AliceO2Group/AliceO2/pull/14322

Reply:
Added check